### PR TITLE
gui: Improve date format.

### PR DESCRIPTION
**Pull request by Croden1999**
# About
Improve date format in other languages (e.g. ru, jp, ko, zh and ch), looks like same the PS Vita. The code should be checked to ensure that it correctly recognizes the correct date.

# Result
image

### CODE REVIEW

File: lang/ru.xml [hunk 1]
```diff
@@ -64,32 +64,65 @@
...
+			<month>марта,</month>
+			<month>апреля,</month>
+			<month>мая,</month>
+			<month>июня,</month>
+			<month>июля,</month>
+			<month>августа,</month>
+			<month>сентября,</month>
+			<month>октября,</month>
+			<month>ноября,</month>
+			<month>декабря,</month>
```
Comment (Zangetsu38) [inline lang/ru.xml:125]: why russian have , ? you have screen for it ? 🤔

**Comment by Zangetsu38** [comment]
goot catch for days on lock screen, i have no thinks asian can haven things here

File: lang/ru.xml (hunk 1 above)
Comment (Croden1999) [reply lang/ru.xml:125]: Wait a minute, Mr. Zangetsu. i took a screenshot in my PSVita.

**Comment by Croden1999** [comment]
> goot catch for days on lock screen, i have no thinks asian can haven things here

**Comment by Zangetsu38** [comment]
> > goot catch for days on lock screen, i have no thinks asian can haven things here
> 
> 

i no view any , ?

File: lang/ru.xml (hunk 1 above)
Comment (Zangetsu38) [reply lang/ru.xml:125]: Your screenshots no show ,  in end month ? 🤔

Comment (Zangetsu38) [reply lang/ru.xml:125]: eu you can check if other disposition havbe this , ? have 3 mode for that

Comment (Zangetsu38) [reply lang/ru.xml:125]: i speek of SCE_SYSTEM_PARAM_DATE_FORMAT

Comment (Croden1999) [reply lang/ru.xml]: > why russian have , ? you have screen for it ? 🤔

ohhh, i found it, it is my wrong.

Comment (Croden1999) [reply lang/ru.xml]: > i speek of SCE_SYSTEM_PARAM_DATE_FORMAT

I really appreciate Mr. Zangetsu that I didn't realize I made such a small mistake.

Comment (Zangetsu38) [reply lang/ru.xml]: Sorry, i have no view, but so, i asked if have , for motn aon 3 mode of date format ?

Comment (Croden1999) [reply lang/ru.xml]: > Sorry, i have no view, but so, i asked if have , for motn aon 3 mode of date format ?

is none, i didn't notice it the first time.

Comment (Zangetsu38) [reply lang/ru.xml]: > > Sorry, i have no view, but so, i asked if have , for motn aon 3 mode of date format ?
> 
> is none, i didn't notice it the first time.

is what ?

Comment (Croden1999) [reply lang/ru.xml]: > is what ?

the ","

Comment (Zangetsu38) [reply lang/ru.xml]: you lost me here 🤔 you have pushed bu code in not changed,for both you change caps month and yes on your psvita screen caps month have , in end

**Review by Zangetsu38 — APPROVED** [review]

File: lang/ru.xml (hunk 1 above)
Comment (Croden1999) [reply lang/ru.xml]: > you lost me here 🤔 you have pushed bu code in not changed,for both you change caps month and yes on your psvita screen caps month have , in end

thx u so mush, Mr. Zangetsu, u help me a lot. 😭

**Pull request merged**